### DISCUSSION TITLE
Exception when setting hyperlink in portion of paragraph of a cell

### DISCUSSION THREAD

**Discussion by YRobles** [ShapeCrawler/ShapeCrawler#394] [General]
Hi,
I'm trying to set up a Hiperlink to a cell's content. But I'm getting this exception:
![image](https://user-images.githubusercontent.com/4245483/203537865-9dea8609-8514-4136-8d36-c401010f61e0.png)

I think this could be for this:
![image](https://user-images.githubusercontent.com/4245483/203538039-ac6fd1f2-1906-4b17-949c-b366ee03e787.png)

Am I doing something wrong or it's a bug?
Thanks in advanced.

**Comment by AdamShakhabov** [comment]
Hello @YRobles. Thank you for the report.

It is a bug. I'm working on fix.

**Comment by AdamShakhabov** [comment]
The fix [**v0.37.1**](https://github.com/ShapeCrawler/ShapeCrawler/releases/tag/v0.37.1) is done. @YRobles, please check your case.
